### PR TITLE
Load SkyX files on iOS where available

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: build/config/ios/ios_app.py [hunk 1]
```diff
@@ -15,9 +15,25 @@
...
+    else:
+      return -1
+
+  return 0
+
 
 def ProcessInfoPlist(args):
   output_plist_file = os.path.abspath(os.path.join(args.output, 'Info.plist'))
+
+  print os.path.dirname(output_plist_file)
```
Comment (eseidelGoogle) [inline build/config/ios/ios_app.py]: Did you mean to leave this in?

File: build/config/ios/rules.gni [hunk 2]
```diff
@@ -131,7 +135,7 @@ template("ios_app") {
   copy_gen_target_name = target_name + "_copy"
   copy(copy_gen_target_name) {
     sources = [
-      "$root_build_dir/Info.plist",
+      "$root_build_dir/plist/$app_name/Info.plist",
```
Comment (eseidelGoogle) [inline build/config/ios/rules.gni]: root_build_dir seems wrong.  That's like out/ios_sim_Debug, I think you want $target_gen_dir or something?

File: sky/shell/BUILD.gn [hunk 3]
```diff
@@ -167,10 +167,40 @@ if (is_android) {
   import("//build/config/ios/rules.gni")
   import("//build/config/ios/ios_sdk.gni")
 
+  group("ios_shell_scaffolding") {
```
Comment (eseidelGoogle) [inline sky/shell/BUILD.gn]: Did you mean to leave this?

File: build/config/ios/ios_app.py (hunk 1 above)
Comment (chinmaygarde) [inline build/config/ios/ios_app.py]: No. Fixing.

File: build/config/ios/rules.gni (hunk 2 above)
Comment (chinmaygarde) [inline build/config/ios/rules.gni]: Does it matter if we are still giving the intermediate plist files from each target their own directory?

File: sky/shell/BUILD.gn (hunk 3 above)
Comment (chinmaygarde) [inline sky/shell/BUILD.gn]: No. Fixing.

File: build/config/ios/rules.gni [hunk 4]
```diff
@@ -115,14 +119,14 @@ template("ios_app") {
     script = ios_app_script
 
     sources = [ invoker.info_plist ]
-    outputs = [ "$root_build_dir/Info.plist" ]
+    outputs = [ "$root_build_dir/plist/$app_name/Info.plist" ]
```
Comment (eseidelGoogle) [inline build/config/ios/rules.gni]: still not a big fan of adding a new top level build directory, but we can fix this in another pass.

**Comment by eseidelGoogle** [comment]
lgtm

**Comment by eseidelGoogle** [comment]
Fixes https://github.com/domokit/sky_engine/issues/252

**Pull request merged**